mlir/unittests/Analysis/Presburger/CMakeLists.txt

```diff
@@ -1,8 +1,10 @@
 add_mlir_unittest(MLIRPresburgerTests
   MatrixTest.cpp
   SimplexTest.cpp
+  ParserTest.cpp
 )
 
 target_link_libraries(MLIRPresburgerTests
-  PRIVATE MLIRPresburger)
+  PRIVATE MLIRPresburger
+  MLIRLoopAnalysis)
```

Comment:
Wait, these are the dependencies for MLIRPresburgerTests, not MLIRPresburger. My bad, there is no cyclic dependency here then